.github/workflows/ruby-ci.yml

```diff
@@ -0,0 +1,35 @@
+name: Ruby-CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  ruby-tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        name: Checkout
+
+      - uses: ruby/setup-ruby@v1
+        name: Use Ruby 2.6
+        with:
+          ruby-version: 2.6
```

Comment:
2.7.2 is the latest: https://www.ruby-lang.org/en/downloads/releases/. Can we test on 2.7 instead? Ideally we should cover both 2.6 _and_ 2.7, but that can be done later.

Reply:
Actually can you use a matrix like the node workflow?
```
strategy:
  matrix:
    ruby-version: [2.6.x, 2.7.x]
```

Reply:
If you switch to the matrix, don't forget to also change the shipit check :)